runelite-client/src/main/java/net/runelite/client/plugins/hunter/HunterConfig.java

```diff
@@ -69,6 +71,18 @@ default Color getEmptyTrapColor()
 		return Color.RED;
 	}
 
+	@Alpha
+	@ConfigItem(
+			position = 4,
```

Comment:
The config items after this one should have their positions updated. (I'm aware the sorting has them in the intended order, but it helps to be explicit when using position-based sorting)